cyclonedx-bom/src/models/component.rs

```diff
@@ -45,7 +45,8 @@ pub struct Component {
     pub publisher: Option<NormalizedString>,
     pub group: Option<NormalizedString>,
     pub name: NormalizedString,
-    pub version: NormalizedString,
+    // todo: 1.3 vs. 1.4 - this field is an Option in 1.4
+    pub version: Option<NormalizedString>,
```

Comment:
Is there anything to be done here? I don't think it should impact this struct. 
Maybe validate that this field is present in the 1.3 parser but not 1.4?